3-box-model/style.css

```diff
@@ -0,0 +1,35 @@
+
+footer {
+    background: black;
+    border: 2px solid orange;
+    bottom: 0;
+    color: white;
+    left: 0;
+    position: fixed;
+    right: 0;
+}
+
+header {
+    background: black;
+    border: 2px solid green;
+    color: white;
+    position: sticky;
```

Comment:
this element is not being visually sticky, can you guess what you're missing to make it work? clue: it's a top property.